sc2/bot_ai.py

```diff
@@ -588,6 +601,11 @@ def _prepare_step(self, state):
         self.supply_cap: Union[float, int] = state.common.food_cap
         self.supply_used: Union[float, int] = state.common.food_used
         self.supply_left: Union[float, int] = self.supply_cap - self.supply_used
+
+        # Workaround Zerg supply rounding bug
+        if self.race == Race.Zerg:
+            self._correct_zerg_supply()
```

Comment:
Maybe be super smart and also move `self.larva_count: int = state.common.larva_count` here? :)

Reply:
That feels like a separate PR to me...

Reply:
That is the difference between you and me :p